src/app/listing/courses-list/courses-list.component.spec.ts

```diff
@@ -22,4 +25,16 @@ describe('CoursesListComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should create <app-courses-list-item> for each item in courses', () => {
+    const courses = [{ id: 1, }, { id: 1 }];
```

Comment:
Should be full courses object, this is failing in compilation phase 
Type '{ id: number; }' is not assignable to type 'CoursesListItem'.